.packit.yaml

```diff
@@ -30,6 +30,9 @@ jobs:
       - epel-8-x86_64
       - epel-9-x86_64
       - epel-10-x86_64
+      - epel-8-aarch64
```

Comment:
You can discard epel-8. We will drop that soon

Reply:
alirght, thanks